stock_landed_costs_mrp_subcontracting/models/stock_landed_cost.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 ForgeFlow, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from collections import defaultdict
+
+from odoo import models
+
+
+class StockLandedCost(models.Model):
+    _inherit = "stock.landed.cost"
+
+    def _get_targeted_move_ids(self):
+        moves = super(StockLandedCost, self)._get_targeted_move_ids()
+        moves_to_remove = self.env["stock.move"]
+        moves_to_add = self.env["stock.move"]
+        for move in moves:
+            mo = move.move_orig_ids.production_id[-1:]
+            if mo and mo.picking_type_id.code == "mrp_operation":
+                moves_to_remove += move
+                moves_to_add += move.move_orig_ids
+        moves -= moves_to_remove
+        moves += moves_to_add
+        return moves
+
+    def _compute_allowed_picking_ids(self):
```

Comment:
```suggestion
    def _compute_allowed_picking_ids(self):
```

sorry for the noise, but since `_compute_allowed_picking_ids` has been removed from this commit https://github.com/odoo/odoo/commit/f329de26